text/metrics/0156-counter-gauge-model.md

```diff
@@ -0,0 +1,356 @@
+  speedGauge.Set(wheelSpeed[i], { 'wheel': i })
+}
+```
+
+This form of Gauge rewrite is generally useful to capture additional
+measurements by creating distinct metric streams.
+
+## Meaning-preserving attribute erasure
+
+Several rules for rewriting metric events that preserve meaning have
+been shown above, focused on introducing new attributes and new
+measurements in ways do not change existing meaning or alter existing
+interpretations.
+
+Removing attributes from metric events does not, by definition, change
+their meaning, since attributes are interpreted as event selectors.
+Removing attributes from aggregated streams of OpenTelemetry Metrics
+data requires attention to the meaning being conveyed.
+
+Safe attribute erasure for OpenTelemetry Metrics streams is specified
+in a way that preserves meaning while removing only the forms of
+interpretation that made use of the erased attribute.
+
+_Reaggregation_ describes the process of combining OpenTelemetry
+metric streams.  For reaggregation to preserve meaning, Sum points
+must be combined by adding the inputs and Gauge points must be
+combined by selecting the last or random value.
+
+Note that erasure of attributes is defined so that it reverses the
+effect of introducing new measurements, and meaning is preserved in
+both directions.  This explains the definition for default
+aggregations that should be applied when re-aggreation OpenTelemetry
+metrics streams.  Sum streams are re-aggregated to preserve the
+implied rate, while Gauge points are reggregated to preserve the
+implied distribution of individual values.
+
+## Conveying meaning to the user
+
+OpenTelemetry states a requirement separating the API from the
+implementation, and to do so we have defined the meaning of metrics
+API events.  To preserve meaning through stages of reaggregation, we
+have specified distinct default aggregation rules for Counter and
+Gauge streams.
+
+When attributes are used with Counter and Gauge instruments, every
+distinct combination of attribute values determines a separate
+OpenTelemetry metrics stream, and each stream conveys meaning
+independently.  Because meaning is independent from the attributes
+used, the user may wish to disregard some attributes when interpreting
+a stream of metrics, restricting their attention to a subset of
+attributes.
+
+In database systems, this process is refered to as a performing a
+"Group-By", where aggregation is used to combine streams within each
+distinct set of grouped attributes.  For the benefit of OpenTelemetry
+users, Metrics systems are encouraged to choose a a meaning-preserving
+aggregation when grouping metric streams to convey meaning to the
+user.
+
+When conveying meaning to the user by grouping and aggregating over a
+subset of attribute keys, the default aggregation selected should be
+one that preserves meaning.  For monotonic Counter instruments, this
+means conveying the combined rate of each group.  For UpDownCounter
```

Comment:
Why does monotonic counter instruments turn into a rate?  Did I miss an above description on how UpDown vs. Counter have different aggregation meaning?

Reply:
Maybe this [description](https://github.com/jmacd/oteps/blob/jmacd/countergauge/text/metrics/0156-counter-gauge-model.md#meaning-and-interpretation-of-counter-and-updowncounter-events)?